config/routes.rb

```diff
@@ -80,6 +80,9 @@
       end
 
       resources :remote_execution_features, :only => [:show, :index, :update]
+
+      get 'cockpit_redirect', to: 'remote_execution_features#cockpit_redirect'
```

Comment:
I could only make this work as `/api/cockpit_redirect`. not as `/cockpit_redirect` as we had before.  This is not really a API entry point, and it should, for example, behave differently when called with an expired session token.  As part of `/api`, the request will be rejected with 401, but as a regular web ui path, it would redirect to the login page.  The latter is more appropriate for this, I'd say.

Reply:
When putting this outside the `/api` block, the error is:

```
The action 'cockpit_redirect' could not be found for RemoteExecutionFeaturesController
```

Reply:
I've update this to `cockpit/redirect` and works for me